### PR TITLE
feat: add web adaptor

### PR DESCRIPTION
**Pull request by aphilobos**
**Related Issue:** #

## Summary
**added**
web-adaptor-16
web-adaptor-24
web-adaptor-32

file_05_screenshot

### CODE REVIEW

File: packages/calcite-ui-icons/docs/keywords.json [hunk 1]
```diff
@@ -8860,5 +8860,80 @@
...
+      "format",
+      "justification"
+    ],
+    "category": "Lists",
+    "release": "3.29.6"
+  },
+  "web-adaptor": {
+    "alias": ["3.29.7", "gis", "core", "web", "diagram", "nodes", "gis", "storage", "servers", "transfer"],
+    "category": "GIS",
+    "release": "3.29.7"
```
Comment (benelan) [inline packages/calcite-ui-icons/docs/keywords.json:8937]: The version here is incorrect, it will be released as `3.32.0`. For reference, the versioning strategy is called [`semver`](https://semver.org/). 

TLDR; The format is `<MAJOR>.<MINOR>.<PATCH>`

- When changing an existing icon, use the `fix:` prefix in the PR title and increment the `<PATCH>` number.

- When you add icons, use the `feat:` prefix in the PR title, increment the `<MINOR>` version, and set the `<PATCH>` number to `0`.

- When introducing a change that breaks previous functionality (e.g. renaming icons), use the `refactor!:` prefix, increment the `<MAJOR>` number, and set the `<MINOR>` and `<PATCH>` versions to `0`.

Please let me, @jcfranco, and @geospatialem know before installing breaking changes, because they require advanced messaging to our users.

File: packages/calcite-ui-icons/docs/keywords.json [hunk 2]
```diff
@@ -8860,5 +8860,80 @@
...
+  },
+  "connection-to-connection": {
+    "alias": ["3.29.5", "generic", "connect", "two points", "line"],
+    "category": "Generic",
+    "release": "3.29.5"
+  },
+  "connection-middle": {
+    "alias": ["3.29.5", "generic", "line", "point", "middle", "connect"],
+    "category": "Generic",
+    "release": "3.29.5"
```
Comment (benelan) [inline packages/calcite-ui-icons/docs/keywords.json:8917]: The versions starting at `3.29.1` are also incorrect. Can you please create a separate `fix` PR that updates them to the correct versions? You can see which version the icons were released as by checking the [changelog](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-ui-icons/CHANGELOG.md)

**Review by benelan — CHANGES_REQUESTED** [review]
